MessageCrackWiz.vcxproj

```diff
@@ -13,17 +13,17 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{B0FB4944-A964-4C5C-96D6-72B568F2D938}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>

```

Comment:
Do not change default build tools.
New platforms must be added separately, or we must migrate to other build system such as CMake.